core/src/main/resources/db/changelog/changesets/2023_10_03_sendgrid_event.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: "sendgrid_event"
+      author: mbemis
+      changes:
+        - createTable:
+            tableName: sendgrid_event
+            columns:
+              - column: { name: id, type: uuid, defaultValueComputed: gen_random_uuid(), constraints: { nullable: false, primaryKey: true } }
+              - column: { name: msg_id, type: text, constraints: { nullable: false, unique: true } }
+              - column: { name: subject, type: text }
+              - column: { name: to_email, type: text }
```

Comment:
wondering if `to_email` should be a FK.

Additionally, in my experience so far, none of these nullable fields have ever been null in the API response, but I'm still hesitating to assume that will always be the case